### DISCUSSION TITLE
Ideas for future development

### DISCUSSION THREAD

**Discussion by blyamur** [metaladmiral/Foodee#3] [Ideas]
In order to have the ability to scale and develop the service, initially lay down the multilingual option. For example, a breakdown by region, country, and language support separately for each region. Even if it is not used now, it can save a lot of time and effort in the future.

**Comment by metaladmiral** [comment]
Hey, I do have some previous experience in building multilingual apps. But, I have built this project for Indian food recommendations only (for now) and know what type of people will be it's potential consumer. I am not sure to what extent it would help/what will be it's demand in other countries. What do you think?
